tests/screens/test_sync.py

```diff
@@ -265,6 +227,15 @@ async def test_save_teams(app, git_repository, helpers, mocker):
             """
         )
 
+        assert get_team_members_mock.call_count == 2
+        assert get_team_members_mock.call_args_list[0][0][1] == 'bar-team'
+        assert get_team_members_mock.call_args_list[0][1]['refresh']
+        assert get_team_members_mock.call_args_list[1][0][1] == 'foo-team'
+        assert get_team_members_mock.call_args_list[1][1]['refresh']
+
+        assert get_deactivated_users_mock.call_count == 1
+        assert list(get_deactivated_users_mock.call_args_list[0][0][1]) == ['j', 'jira-foo1', 'jira-bar1']
+
```

Comment:
I would avoid making this sort of assertions, I think they assert the "how" more than the "what".

Reply:
So you would just assert the call count?

Reply:
As discussed offline, I'm simply not a fan in general of checking the calls with such detail because it keeps the coupling between the test and the code; but I'm fine if you see value in it 👍 .